### ISSUE TITLE
upsample and filter numbers

### ISSUE THREAD

**Issue by zymize** [Zheng222/IDN-tensorflow#7]
There is another questions that I would like to consult .
In your distillation model:

1.  "tmp = tf.layers.conv2d(x, 48, (3, 3), (1, 1), padding='same', activation=lrelu, name=name+'/conv1')"

     "tmp2 = tf.layers.conv2d(tmp2, 80, (3, 3), (1, 1), padding='same', activation=lrelu,         name=name+'/conv6')"
How to confirm 48 and 80 filters ? is this a result of continuous experimention?
2."def upsample(x, scale=4, features=32, name=None)"
   "def upsample_fn(x, scale=4)"
In these  function definitions, scale=4 . if scale=2/3/4,Does scale=4 in these  function definitions need to be modified?

**Comment by zymize** [comment]
如果您有时间可以帮忙解答一下吗？
初学超分辨率重建，我的英语不太好，如果我问了一些特别基础的问题，写的不对的地方请谅解。
多次打扰，深感抱歉。

**Comment by Zheng222** [comment]
@zymize 
1. The number of channels is usually a multiple of 2, such as 12, 16, 24, 32, 48, 64, etc. 48, 80 are calculated by Eq. (6) and (7).

2.  In function `def upsample_fn(x, scale=4)`, scale can be 2, 3, 4 without modifying any code. I suggest you learn the basics of python and tensorflow 1.x, and then read through the paper and code, it should be helpful to you.
